titanic.ipynb

```diff
@@ -5,14 +5,14 @@
       "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    # Set the dark theme for the plot using matplotlib
</pre><p>test</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/smith-kyle/notebook-diff-tests/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>